llm/vicuna/serve.yaml

```diff
@@ -14,8 +14,6 @@ setup: |
   pip install torch==1.12.1+cu113 --extra-index-url https://download.pytorch.org/whl/cu113
   pip install git+https://github.com/lm-sys/FastChat.git
 
-  echo "Downloading model..."
-  python -u -c "from transformers import AutoModelForCausalLM; AutoModelForCausalLM.from_pretrained('lmsys/vicuna-${MODEL_SIZE}b-v1.3')"
```

Comment:
we don't want to remove this right?

Reply:
Should be fine to remove, as we will wait for the download to finish now in the later run section (This line is a remnant)